templates/head-meta.html

```diff
@@ -10,7 +10,7 @@
 
 <link rel="publisher" href="https://plus.google.com/107187849809354688692">
 <link rel="shortcut icon" href="/images/logos/p-logo-32.png">
-<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Source+Code+Pro:400,700|Roboto+Slab:400,700">
+<link rel="preload" href="//fonts.googleapis.com/css?family=Source+Code+Pro:400,700|Roboto+Slab:400,700">
```

Comment:
Seems like you don't actually load this resource anywhere else. Remove? (Some of the code is in Source Code Pro, but that seems to be already provided by the system)